src/VisualStudio/Core/Def/Implementation/LanguageClient/AlwaysActivateInProcLanguageClient.cs

```diff
@@ -36,26 +38,40 @@ internal class AlwaysActivateInProcLanguageClient : AbstractInProcLanguageClient
             CSharpVisualBasicRequestDispatcherFactory csharpVBRequestDispatcherFactory,
             VisualStudioWorkspace workspace,
             IAsynchronousOperationListenerProvider listenerProvider,
-            ILspWorkspaceRegistrationService lspWorkspaceRegistrationService)
+            ILspWorkspaceRegistrationService lspWorkspaceRegistrationService,
+            DefaultCapabilitiesProvider defaultCapabilitiesProvider)
             : base(csharpVBRequestDispatcherFactory, workspace, diagnosticService: null, listenerProvider, lspWorkspaceRegistrationService, diagnosticsClientName: null)
         {
             _globalOptionService = globalOptionService;
+            _defaultCapabilitiesProvider = defaultCapabilitiesProvider;
         }
 
         public override string Name => "C#/Visual Basic Language Server Client";
 
         protected internal override VSServerCapabilities GetCapabilities()
-            => new VSServerCapabilities
+        {
+            var serverCapabilities = new VSServerCapabilities();
+
+            // If the LSP editor feature flag is enabled advertise support for LSP features here so they are available locally and remote.
+            var isLspEditorEnabled = Workspace.Services.GetRequiredService<IExperimentationService>().IsExperimentEnabled(VisualStudioWorkspaceContextService.LspEditorFeatureFlagName);
+            if (isLspEditorEnabled)
+            {
+                serverCapabilities = _defaultCapabilitiesProvider.GetCapabilities();
```

Comment:
1.  should the default capbilities include TextDcoumentSyntax?  if so, we can remove the code below, and only add it to the case where the experiment is not active.  If not... why not?, should they?

Reply:
yeah they actually already had it - I believe I started this PR before there were there and forgot to remove them.  Updated!